packages/plugin-github-actions/index.js

```diff
@@ -8,6 +9,18 @@ export default {
       return '';
     }
 
+    if (target.startsWith('.')) {
+      const [, org, repo, , sha] = path.split('/');
+
+      return `{BASE_URL}/${join(org, repo, 'tree', sha, target)}`;
```

Comment:
I have another PR I'm working on for Dart or C/C++ that does this same thing. I'll make sure to move this out to a helper when I get back to it.

Reply:
I used `join()` for these because that's how it was done here and it seemed a little cleaner. Do you have a preference to how these are built?

https://github.com/OctoLinker/OctoLinker/blob/a36728e8b29f3d54a1770585041c331d262cc120/packages/resolver-relative-file/index.js#L4

Reply:
I don't mind, but I agree, your solution is more explicit.